main.go

```diff
@@ -154,7 +157,15 @@ func main() {
 		os.Exit(1)
 	}
 
-	kongCtx.FatalIfErrorf(kongCtx.Run(ctx, client))
+	err = kongCtx.Run(ctx, client)
+	if err != nil {
+		if k8serrors.IsForbidden(err) && !nctl.Verbose {
+			kongCtx.Fatalf("Permission denied: are you part of the organization?\nuse --verbose to see detailed error")
```

Comment:
I'd do a

```go
if k8serrors.IsForbidden(err) && !nctl.Verbose {
err = errors.New("Permission denied: are you part of the organization?")
}
kongCtx.FatalIfErrorf(err)
```

I'd do this because you don't see the `else` keywork in golang code that often. You can also check https://go.dev/wiki/CodeReviewComments#indent-error-flow for an explanation of this.